src/main/java/htsjdk/variant/variantcontext/writer/AsyncVariantContextWriter.java

```diff
@@ -53,4 +53,10 @@ public void writeHeader(final VCFHeader header) {
     public boolean checkError() {
         return false;
     }
+
+    @Override
+    public void setVcfHeader(VCFHeader header) {
```

Comment:
The existing code isn't consistent about this, but we generally mark arguments as final in new code. Thanks.